index.html

```diff
@@ -25,7 +26,7 @@
 			});
 		});
 	</script>
-	
```

Comment:
ignore all these white space revisions. They just happened automatically and aren't important.